jenkins-design-language/src/js/components/material-ui/svg-icons/action/accessibility.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import pure from 'recompose/pure';
```

Comment:
I believe this functionality is already in React for function components, and possibly also if you explicitly `class MyComponent extends PureComponent`